Formula/r/rake-completion.rb

```diff
@@ -1,22 +1,22 @@
 class RakeCompletion < Formula
   desc "Bash completion for Rake"
-  homepage "https://github.com/JoeNyland/rake-completion"
-  url "https://github.com/JoeNyland/rake-completion/archive/refs/tags/v1.0.1.tar.gz"
-  sha256 "085801e62cb240311d77885778a603f649b3fd5d85ee279691d1d00bc060bba6"
+  homepage "https://github.com/mernen/completion-ruby"
+  url "https://github.com/mernen/completion-ruby/archive/refs/tags/v1.0.0.tar.gz"
+  sha256 "ee8b84d7ac7444a7388e58a406af56dc0b690a57faa7bcfa4c10671deb788991"
   license "MIT"
+  version_scheme 1
```

Comment:
we try to avoid version_scheme, it might be better to ask if he can make another release?

Reply:
(Repo author here) Looking at the diff, it'd have to be a v1.0.2, right?

Reply:
I copied the version scheme over because the other completion formulae had it. Though tbqh, I wasn't even sure what it was for.

I also considered using rake-completion@2 but it isn't really a parallel version. Nor is it a "fork " so rake-version2 felt weird also.

But since the original has been disabled I thought simple replacement was acceptable. (Particularly because there isn't any api to be concerned with compatibility)

Though if upstream bumped to 1.0.2 that would indeed smooth things.

Reply:
> I copied the version scheme over because the other completion formulae had it. Though tbqh, I wasn't even sure what it was for.

it is because there is no tag in that repo before, and we used calver somehow, and version_scheme was added.

see discussions in https://github.com/Homebrew/homebrew-core/pull/157286

Reply:
good morning! yep, I was looking through the other completions' histories. They were explicitly marked as "version 2" prior to pointing to the 1.0.0 tag.

Removed the version_scheme!

Reply:
@mernen now we just a higher version to ship the change.

Reply:
Alright, v1.0.2 is out!

Reply:
thanks @mernen!

updated formula